ksql-engine/src/test/java/io/confluent/ksql/embedded/KsqlContextTestUtil.java

```diff
@@ -48,10 +50,18 @@ public static KsqlContext create(
   ) {
     final KafkaClientSupplier clientSupplier = new DefaultKafkaClientSupplier();
 
-    final Admin adminClient = clientSupplier
-        .getAdmin(ksqlConfig.getKsqlAdminClientConfigProps());
+    ClientContextFactory clientContextFactory = new ClientContextFactory(
```

Comment:
You should be able to revert the changes in this file with my suggested approach